lara-typescript/src/plugin-api/types.ts

```diff
@@ -40,6 +40,20 @@ export interface IPluginRuntimeContext {
   getFirebaseJwt: (appName: string) => Promise<IJwtResponse>;
   /** Wrapped embeddable runtime context if plugin is wrapping some embeddable. */
   wrappedEmbeddable: IEmbeddableRuntimeContext | null;
+  /**
+   * Logs event to the CC Log Server. Note that logging must be enabled for a given activity.
+   * Either by setting URL param logging=true or by enabling logging in Portal.
+   * ```
+   * PluginAPI.log("testEvent");
+   * PluginAPI.log({event: "testEvent", event_value: 123});
+   * PluginAPI.log({event: "testEvent", someExtraParam: 123});
+   * PluginAPI.log({event: "testEvent", parameters: { paramInParamsHash: 123 }});
```

Comment:
I guess this example should be updated now, e.g. `context.log(...)` instead of `PluginAPI`.

Reply:
Great comments. Thanks.